chisel.yaml

```diff
@@ -1,7 +1,44 @@
-format: chisel-v1
+format: chisel-v2
 
 archives:
     ubuntu:
         version: 22.04
         components: [main, universe]
         suites: [jammy, jammy-security, jammy-updates]
+        public-keys: [ubuntu-archive-key-2018]
+
+public-keys:
+    # Ubuntu Archive Automatic Signing Key (2018) <ftpmaster@ubuntu.com>
+    # rsa4096/f6ecb3762474eda9d21b7022871920d1991bc93c 2018-09-17T15:01:46Z
```

Comment:
What does this line mean? What is the `rsa4096/...` identifier?

Reply:
It's a 4096 bit RSA key with fingerprint `f6ecb3762474eda9d21b7022871920d1991bc93c` and created at time `2018-09-17T15:01:46Z`.
Let me know if you want me to expand it in comments as this.